README.md

```diff
@@ -1,2 +1,188 @@
+                 │                           │  
+                 │  ┌─────────────────────┐  │   ┌─────────────────────┐       ┌─────────────────────┐
+                 └─▶│    RouterClient     │◀─┴──▶│     RouterCache     │◀─────▶│      RouterFor      │
+                    └─────────────────────┘      └─────────────────────┘       └─────────────────────┘
+                               ▲                                                          │  
+                               │                                                          │  
+                               ▼                                                          │  
+                    ┌─────────────────────┐                                               │  
+                    │      RpcClient      │◀──────────────────────────────────────────────┘  
+                    └─────────────────────┘  
+                               ▲  
+                               │  
+                               ▼  
+                    ┌─────────────────────┐  
+                    │  Default gRPC impl  │  
+                    └─────────────────────┘  
+                               ▲  
+                               │  
+           ┌───────────────────┴ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ┐  
+           │                            │  
+           ▼                            ▼                            ▼  
+┌─────────────────────┐      ┌─────────────────────┐      ┌─────────────────────┐  
+│   CeresDBx Node1    │      │   CeresDBx Node2    │      │         ...         │  
+└─────────────────────┘      └─────────────────────┘      └─────────────────────┘  
+```
+
+## Requirements
+- Java 8 or later is required for compilation 
+
+
+## Create table
+CeresDB is a Schema-less time-series database, so creating table schema ahead of data ingestion is not required (CeresDB will create a default schema according to the very first data you write into it). Of course, you can also manually create a schema for fine grained management purposes (eg. managing index).
+
+The following table creation statement（using the SQL API included in SDK ）shows all field types supported by CeresDB：
+
+```java
+SqlResult result = client.management().executeSql("CREATE TABLE MY_FIRST_TABL(" +
+    "ts TIMESTAMP NOT NULL," +
+    "c1 STRING TAG NOT NULL," +
+    "c2 STRING TAG NOT NULL," +
+    "c3 DOUBLE NULL," +
+    "c4 STRING NULL," +
+    "c5 INT64 NULL," +
+    "c6 FLOAT NULL," +
+    "c7 INT32 NULL," +
+    "c8 INT16 NULL," +
+    "c9 INT8 NULL," +
+    "c10 BOOLEAN NULL,"
+    "c11 UINT64 NULL,"
+    "c12 UINT32 NULL,"
+    "c13 UINT16 NULL,"
+    "c14 UINT8 NULL,"
+    "c15 TIMESTAMP NULL,"
+    "c16 VARBINARY NULL,"
+    "TIMESTAMP KEY(ts)) ENGINE=Analytic"
+);
+```
+
+For more examples of table creation statements, see the detail docs [here](./table.md)
+
+
+## Data ingestion example
+```java
+// CeresDBx options
+final CeresDBxOptions opts = CeresDBxOptions.newBuilder("127.0.0.1", 8081) //
```

Comment:
CeresDBx -> CeresDB

Reply:
See the comments above, this will not be modified.